File: core/tmp/Python/math/bulb_switcher_ii.py
def flipLights(n, m):
    n = min(n, 3)
    if m == 0: return 1
    if m == 1: return [2, 3, 4][n - 1]
    if m == 2: return [2, 4, 7][n - 1]
    return [2, 4, 8][n - 1]

if __name__ == "__main__":
    print(flipLights(3, 1))

File: core/tmp/Java/src/com/dbc/algorithm/Math/BulbSwitcherII.java
package com.dbc.algorithm.Math;

public class BulbSwitcherII {
    public static int flipLights(int n, int m) {
        int flag = Math.min(n, 3);
        if (m == 0) return 1;
        if (m == 1) return new int[]{2, 3, 4}[flag - 1];
        if (m == 2) return new int[]{2, 4, 7}[flag - 1];
        return new int[]{2, 4, 8}[flag - 1];
    }
}
